frontend/sharedServices/fallbackImage.ts

```diff
@@ -0,0 +1,14 @@
+import _, {SyntheticEvent} from 'react';
+
+export const fallbackImage = (event: SyntheticEvent<HTMLImageElement, Event>, source: string) => {
+  event.currentTarget.src =
+    source === 'mediaImage'
+      ? `https://s3.amazonaws.com/assets.airy.co/fallbackMediaImage.svg`
+      : `https://s3.amazonaws.com/assets.airy.co/${source}_avatar.svg`;
+
+  if (source === 'mediaImage') {
+    event.currentTarget.style.objectFit = 'contain';
+  }
+
+  event.currentTarget.alt = 'fallback image';
+};
```

Comment:
With @thorstenairy we had the idea of creating _SharedServices_ that contains helper functions needed by both the frontend and our libraries. Here fallbackImage serves for images in both the frontend and the render library.

Reply:
yeah so this was brought up many times and I should probably write down the answer eheh

the tl;dr is "shared" "util" is always bad as then we mix things that have nothing to do with each other. (I think last I talked to @thorstenairy exactly about this because he introduced a shared lib with was a date lib)

A better approach is to have small libraries that solve a specific problem.

to me, it feels like here we're introducting an `avatar library`. So best is 1) call it what it does 2) that way we can move more "avatar related things" there 3) _by far_ themost important point: we do not open the door to catch all libraries

Reply:
@lucapette the idea was to use this function also for the fallback images in the render library. I totally get the point of not falling in 'catch all libraries' so should we create 2 libraries: one `avatar library`  used for the channel avatars (and possibly also the Avatar component), and another library for the fallback images in the render library?

Reply:
duplicating a little of code is better than the wrong abstraction for sure

Reply:
ok we talked offline about it: we do not need to introduce libraries as the abstraction is not worth it. //c @thorstenairy